src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
http://dlss-dev-azaroth.stanford.edu/services/iiif_ns/f1rc/info.json  I believe correctly implements non square tiles at the server side, @jpstroop to confirm?